apps/src/sites/studio/pages/projects/public.js

```diff
@@ -22,21 +22,21 @@ $(document).ready(() => {
     getStore().dispatch(setProjectLists(projectLists));
     const publicGallery = document.getElementById('public-gallery');
     ReactDOM.render(
-      <div>
-        <HeaderBanner
-          headingText={i18n.projects()}
-          subHeadingText={i18n.projectsSubHeading()}
-          short
-        />
-        <StartNewProject
-          isRtl={isRtl}
-          canViewFullList
-          canViewAdvancedTools
-        />
-        <Provider store={getStore()}>
+      <Provider store={getStore()}>
+        <div>
+          <HeaderBanner
+            headingText={i18n.projects()}
+            subHeadingText={i18n.projectsSubHeading()}
+            short
+          />
+          <StartNewProject
+            isRtl={isRtl}
```

Comment:
you may be able to remove isRtl here, now that you're supplying it via redux

Reply:
Fixed in 5e787cb73438028d594ba3e387769280dc454938!